### PR TITLE
Add configuration option to allow TLS cert verification to be disabled

### PR DESCRIPTION
**Pull request by matthew1001**

### CODE REVIEW

**Comment by EnriqueL8** [comment]
Whilst updating the firefly-perf-cli with the mTLS changes I noticed that it using a commit pointing to this branch, so going to review this and see if it conflicts. Spoke to @matthew1001 that he will make the changes to the cli to pick up the latest version

File: pkg/wsclient/wsclient.go [hunk 1]
```diff
@@ -96,6 +98,8 @@ func New(ctx context.Context, config *WSConfig, beforeConnect WSPreConnectHandle
 		wsdialer: &websocket.Dialer{
 			ReadBufferSize:  config.ReadBufferSize,
 			WriteBufferSize: config.WriteBufferSize,
+			//nolint:gosec
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: config.DisableTLSVerification},
```
Comment (EnriqueL8) [inline pkg/wsclient/wsclient.go:102]: Unfortunately this doesn't play nicely with a secure websocket server which is the case when mTLS is enabled.

What we actually want is the ability to provide a `tls` section under the WSClient or take a configuration from elsewhere

**Review by EnriqueL8 — COMMENTED** [review]
I think we should get this in and then raise a separate PR to properly expose a TLS config on this wsclient to secure the HTTP Upgrade.

**Comment by peterbroadhurst** [comment]
Closing this as I believe WS TLS was implemented in #67 
@matthew1001 - please feel to raise a new PR to add a `HTTPConfTLSInsecureSkipVerify` boolean config variable to `fftls` with suitable warnings in the doc-entry for the config var, if there's still a requirement for this (I could see dev testing of WebHook clients from FireFly being an example of a limited use case for disabling TLS cert validation).

**Pull request closed without merging**